Docker/README.md

```diff
@@ -0,0 +1,320 @@
+    RUN echo 'net.ipv4.conf.all.rp_filter=0' >> /etc/sysctl.conf
+    RUN echo 'net.ipv4.ip_forward=1' >> /etc/sysctl.conf
+    RUN echo 'net.ipv6.conf.all.forwarding=1' >> /etc/sysctl.conf
+    RUN git clone git://github.com/OpenOverlayRouter/oor.git
+    WORKDIR /oor
+    RUN make
+    RUN make install
+    RUN rm -rf /oor
+    COPY files/*.conf /etc/
+    COPY files/*.sh /tmp/
+    COPY files/*.yaml /tmp/
+    ENV DEBUG <int 0..3>
+    ENV OPMODE xTR
+    ENV IPMAPRESOLVER <IP of the MapResolver>
+    ENV IPMAPSERVER <IP of the MapServer>
+    ENV KEYMAPSERVER <String>
+    ENV IPPROXYETRV4 <IP of the Proxy ETR IPv4>
+    ENV IPPROXYETRV6 <IP of the Proxy ETR IPv6>
+    ENV IPV4EIDPREFFIX <EID IPv4 Preffix>
+    ENV IPV6EIDPREFFIX <EID IPv6 Preffix>
+    CMD /tmp/start.sh $DEBUG $OPMODE $IPMAPRESOLVER $IPMAPSERVER $KEYMAPSERVER $IPPROXYETRV4 $IPPROXYETRV6 $IPV4EIDPREFFIX $IPV6EIDPREFFIX
+
+So you can Configure your xTR setting properly the values for each variable.
+
+Working with Docker-Compose Stacks
+----------------------------------
+
+The Docker Compose Stack will look like the following:
+
+    version: "3"
+    services:
+      oor:
+        image: jortizpa/oor
+        cap_add:
+          - NET_ADMIN
+          - NET_RAW
+        devices:
+          - "/dev/net/tun:/dev/net/tun"
+        networks:
+          0rloc:
+              ipv4_address: <ip>
+          1eids:
+              ipv4_address: <ip>
+        environment:
+          - IPV4EIDPREFFIX="<network>\/<mask>"
+          - IPV6EIDPREFFIX="<network>\/<mask>"
+          - DEBUG="<int 0..3>"
+          - OPMODE="xTR"
+          - IPMAPRESOLVER=<IP of the MapResolver>
+          - IPMAPSERVER=<IP of the MapServer>
+          - KEYMAPSERVER=<String>
+          - IPPROXYETRV4=<IP of the Proxy ETR IPv4>
+          - IPPROXYETRV6=<IP of the Proxy ETR IPv6>
+          - IPV4EIDPREFFIX=<EID IPv4 Preffix>
+          - IPV6EIDPREFFIX=<EID IPv6 Preffix>
+
+    networks:
+      1eids:
+        driver: macvlan
+        ipam:
+          driver: default
+          config:
+          - subnet: <EID IPv4 Preffix>
+        driver_opts:
+          parent: <linux host interface>
+          macvlan_mode: bridge
+      0rloc:
+        driver: macvlan
+        ipam:
+          driver: default
+          config:
+          - subnet: <RLOC Subnet>
+        driver_opts:
+          parent: <linux host interface>
+          macvlan_mode: bridge
+
+Running Open Overlay Router in Containers
+-----------------------------------------
+
+Using Docker daemon natively:
+
+      docker create --net=<RLOC_Docker_Network_Name> --ip=<IP_RLOC> --name <docker_name> -it --device=/dev/net/tun --cap-add=NET_ADMIN --cap-add=NET_RAW openoverlayrouter/oor:latest 
+      docker network connect <EID_Docker_Network_Name> --ip=<EID_IP_forContainer> <docker_name>
+      docker start <docker_name>
+
+Using Docker Compose (It create docker networks for you):
+
+      docker-compose -f docker-compose-network.yml up -d
+
```

Comment:
I would remove everything below this point, and link to the main ../README.md file, to avoid duplicating text. Or the reason behind leaving things here is that the Docker image that will be pulled is built at a point in time where the below is true, and if the code moves forward, the main documentation may not apply to Docker? If that's the case, this would need to be checked for changes constantly, which is error prone. Ideally a CI job would publish new images whenever there are changes in OOR.